benchmarks/src/main/scala/akka.kafka.benchmarks/KafkaConsumerBenchmarks.scala

```diff
@@ -71,37 +71,40 @@ object KafkaConsumerBenchmarks extends LazyLogging {
 
     var lastProcessedOffset = 0L
     var accumulatedMsgCount = 0L
-    val semaphore = new Semaphore(1)
+    val commitInProgress = new AtomicBoolean(false)
```

Comment:
There is only one thread involved, as far as I understand the Kafka threading model. Callbacks are invoked by the thread invoking `poll`. That would mean that there is no need for atomic.